File: api/ComidaAPI_base/Models/Bebida.cs
using Google.Cloud.Firestore;

namespace ComidaAPI.Models;

[FirestoreData]
public class Bebida
{
    [FirestoreProperty("id")]
    public string Id { get; set; } = string.Empty;

    [FirestoreProperty("userId")]
    public string UserId { get; set; } = string.Empty;

    [FirestoreProperty("nombre")]
    public string Nombre { get; set; } = string.Empty;

    [FirestoreProperty("precio")]
    public float Precio { get; set; }

    [FirestoreProperty("lugar")]
    public string Lugar { get; set; } = string.Empty;
}

File: app/src/main/java/com/example/comida/persistencia/modelos/Bebida.kt
package com.example.comida.persistencia.modelos

data class Bebida(
    val id: String = "",
    val userId: String = "",
    val nombre: String = "",
    val precio: Float = 0f,
    val lugar: String = ""
) {
    fun estaEnPresupuesto(presupuesto: Float) = precio <= presupuesto
}